core/src/main/scala/com/salesforce/op/OpWorkflow.scala

```diff
@@ -514,7 +517,8 @@ class OpWorkflow(val uid: String = UID[OpWorkflow]) extends OpWorkflowCore {
     maxJSDivergence: Double = 0.90,
     maxCorrelation: Double = 0.95,
     correlationType: CorrelationType = CorrelationType.Pearson,
-    protectedFeatures: Array[OPFeature] = Array.empty
+    protectedFeatures: Array[OPFeature] = Array.empty,
+    textBinsFormula: (Summary, Int) => Int = RawFeatureFilter.textBinsFormula
```

Comment:
why the int?

Reply:
it's the bins.

Reply:
Yeah but that is not what we want the formula to look like long term

Reply:
so what is your suggestion on the inputs / outputs then?